### ISSUE TITLE
Add Contributor

### ISSUE THREAD

**Issue by rajat-smpx** [rajat-smpx/Contributors#4]
Add a new row in contributors table.
Open a pull request.

**Comment by AlexAlper** [comment]
Why do this?

**Comment by bcye** [comment]
You mean adding yourself to the contributors table?

**Comment by AlexAlper** [comment]
I can not understand why add myself?
Where will I participate?

**Comment by bcye** [comment]
@AlexAlper I think it’s meant to cheat yourself a pull request

**Comment by rajat-smpx** [comment]
@AlexAlper This is just to make beginners familiar with Pull Requests.

**Comment by rajat-smpx** [comment]
@bcye Yes, you just have to add yourself as contributor in README file.

**Comment by bcye** [comment]
@rajat-smpx oh sorry then for suggesting that this repo is for cheating
